### DISCUSSION TITLE
Images not pulling

### DISCUSSION THREAD

**Discussion by Setman590** [tltneon/lgsl#59] [General]
Hi,

I am trying to add the sidebar banners to my site. When I add the code: <a href="http://darkwarriors.me/lgsl/?s=1"><img src="http://darkwarriors.me/lgsl/userbar.php?s=1" /></a>
It adds the long thin banner, but I want to use the box style banner, like the one in the Using Zones in the wiki
What am I doing wrong?

Thanks

**Comment by tltneon** [comment]
Hi @Setman590,

Some time ago ther was added new box style banner. For use it you need to place update LGSL up to v6.1.1
